modules/mono/editor/Godot.NET.Sdk/Godot.SourceGenerators/Common.cs

```diff
@@ -384,5 +384,95 @@ ISymbol exportedMemberSymbol
                 typeArgumentSyntax.GetLocation(),
                 typeArgumentSyntax.SyntaxTree.FilePath));
         }
+
+        public static readonly DiagnosticDescriptor GlobalClassMustImplementGodotObjectRule =
```

Comment:
nit-pick: I like to use the word _implement_ with interfaces and _derive_ with inheritance.

```suggestion
        public static readonly DiagnosticDescriptor GlobalClassMustDeriveFromGodotObjectRule =
```